DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Helpers/ParameterValueExtractor.cs

```diff
@@ -20,6 +20,11 @@ public ParameterValueExtractor(ToSpeckleScalingService scalingService)
 
   public object? GetValue(Parameter parameter)
```

Comment:
I could be missing something, but I presume we cannot be more specific than object and there's no way to remove the nullable?

Reply:
Yeah, we are already being more specific. 

If you look at this method, you'll see that it is basically just a switch on the parameterType which then dispatches the appropriate, strongly typed, method. These are public methods, so if you know that a parameter is going to be a double, then you can call GetValueAsDouble().

I don't think we need to worry about the nullability. We've had issues in the past with sending default values in the past (i.e. sending unset parameters with a value of 0 instead of a value of null)